### PR TITLE
[PREVIEW] SL-1503 Assign joh or room via planner

### PR DESCRIPTION
**Pull request by zielinskikamil**
Added Summary message + its service + unit tests
## This PR needs to be merged in order to test rollback mechanism https://github.com/hmcts/snl-events/pull/120

### CODE REVIEW

File: src/app/planner/services/summary-message.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+              StoreModule.forFeature('judges', judgesReducers.reducers),
+            //   StoreModule.forFeature('rooms', roomReducers.reducers)
+            ],
+            providers: [
+                SummaryMessageService,
+            ]
+          });
+          summaryMessageService = TestBed.get(SummaryMessageService);
+          store = TestBed.get(Store);
+          store.subscribe(console.log)
```
Comment (leszekgu) [inline src/app/planner/services/summary-message.service.spec.ts]: Is this console.log and commented line bellow still needed?

File: src/app/planner/services/summary-message.service.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                this.buildUnassignedMsg(resourceType, event.detail.event) :
+                this.buildAssignedMsg(resourceType, resourceId)
+        }
+    }
+
+    private isTimeChanged(event): boolean {
+        return event.detail.duration.asMilliseconds() > 0
+    }
+
+    private isResourcedChanged(event, resourceType, resourceId): boolean {
```
Comment (leszekgu) [inline src/app/planner/services/summary-message.service.ts]: a typo or there is a reason for 'isResourceDChanged' ( why D?)

File: src/app/core/callendar/transformers/default-data-transformer.ts [hunk 3]
```diff
@@ -15,6 +15,8 @@ export class DefaultDataTransformer implements IcalendarTransformer<SessionViewM
         const sessionType = (session.sessionType) ? session.sessionType.description : 'No Session type';
 
         return {
+            person: session.person,
```
Comment (leszekgu) [inline src/app/core/callendar/transformers/default-data-transformer.ts]: Correct me if I'm wrong, but are we updating this fields after a resource has been changed?
for example Session A, is in Room1, we move it to Room2, now besides having proper resourceId in this transformed fullcalendar element are those fields also updated?

Comment (zielinskikamil) [reply src/app/core/callendar/transformers/default-data-transformer.ts]: yep, all events are rerender and has proper values :)

**Pull request merged**